src/nimlsp.nim

```diff
@@ -481,7 +488,7 @@ while true:
                 file.writeLine line
               file.close()
             debugEcho "fileuri: ", fileuri, ", project file: ", openFiles[fileuri].projectFile, ", dirtyfile: ", filestash
-            let diagnostics = getNimsuggest(fileuri).chk(fileuri[7..^1], dirtyfile = filestash)
+            let diagnostics = getNimsuggest(fileuri).chk(fileuri[10..^1], dirtyfile = filestash)
```

Comment:
Is this a bug? You for some reason changed this from 7 to 10 and not to `uriToPath` which uses 7 or 8 depending on OS.

Reply:
Oops! My bad, sorry. It's a leftover after debugging.

Reply:
Fixed.